game/pokemon.py

```diff
@@ -17,16 +19,21 @@ def __init__(
         name: str,
         dex_number: int,
         types: List[Element],
-        total_health: Optional[int] = None
+        total_health: Optional[int] = None,
     ):
+        """Property of Pokemon initialized"""
```

Comment:
It's actually properties and not a single property, but actually I think docs on `__init__()` methods are kind of redundant in general so you can remove them

Reply:
Removed this comment